core/src/main/resources/hudson/Messages_pl.properties

```diff
@@ -1,6 +1,6 @@
 # The MIT License
 #
-# Copyright (c) 2016, Damian Szczepanik
+# Copyright (c) 2004-2016, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
I would keep the original version since you're the main contributor of this translation file